todo/src/db/localStorage.js

```diff
@@ -0,0 +1,19 @@
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem("state");
+    if (serializedState === null) {
+      return undefined;
```

Comment:
I think you probably need to return an empty array or some dummy state here, when I load it with empty local storage it crashes as the reducer can't return undefined